AlphaWallet/TokenScriptClient/Models/EventsActivityDataStore.swift

```diff
@@ -48,6 +48,15 @@ class EventsActivityDataStore: EventsActivityDataStoreProtocol {
         cachedRecentEventsSubscribable[subscription] = nil
     }
 
+    func getRecentEventsSortedByBlockNumber(forContract contract: AlphaWallet.Address, server: RPCServer, eventName: String, interpolatedFilter: String) -> Results<EventActivity> {
```

Comment:
Should we add a function comment that sorting by date implies sorting by block number?

Reply:
in this case we can update it with replacing `.sorted(byKeyPath: "date", ascending: false)` with `.sorted(byKeyPath: "blockNumber")`

Reply:
Oh yeah, probably faster too. Remember `ascending: false`.